_episodes/00-Overview.md

```diff
@@ -42,64 +42,62 @@ software to analyze data, writing custom code or editing documents.
-topics seem independent but also very much related, so it is likely
-that you might have some familiarity with all of them, happen you know
-well one of them.  If all those topics not familiar to you at all,
-this may take you a long time, for instance 5 to 7 full days. If you
-have good familiarity, some of the information will be already
-partially known and it may take you a few hours to go in detail
-through this material.  In each lesson we provide some estimates for
-time it would take to go through the lesson having an assumption that
-you have some basic familiarity with the topic.
-
-### How acquired knowledge should get used?
+topics seem independent but are also very much related, so it is likely
+that you might have some familiarity with all of them, or that you know
+just one of them well.  If all topics are not familiar to you at all,
+this may take you a long time: for instance, 5 to 7 full days. If you
+are familiar, some of the information may be redundant and it may take
+you only a few hours to go through this material in detail. In each
+lesson, we provide an estimate for the time it would take to learn
+the lesson, assuming you have a basic understanding of the topic.
+
+### How should the acquired knowledge be used?
 
 It is important to **apply** the knowledge gained from the lessons to
-your day-to-day activities as soon as feasible: Start using shell
-and/or do it more efficiently (use shortcuts, start scripting, make
-those scripts robust, etc.), use version control systems for anything
+your day-to-day activities as soon as possible! Start using shell
+and/or do it more efficiently by using shortcuts, scripting, making
+those scripts robust, etc. Use version control systems for anything
 you change (code, data, documents) and increase "traceable" collaboration
-exchange (even though just between you on computer 1 and you on
-computer 2), get curious and check what you are using and start
-provisioning your own computation environments, report problems you
-run into and do not let them just left unresolved.
-
-One of the efficient learning approaches of the materials could be
-first to skim through all the materials, while only catching the key
-concepts and start applying them right away. Then, having gained some
-experience and having stumbled upon some problems, come back and go
-through the relevant topic in greater detail while concentrating on
-the aspects relevant to your early encounters.
-
-**The least efficient approaches would be to spend a week "learning
-it" only to successfully forget all of it by not using any of the
-learned tools or recommended practices.**
+exchange, even if it’s just between you on computer 1 and you on
+computer 2. Get curious and check what you are using; start
+provisioning your own computation environments. Report problems you
+run into and don’t just leave them unresolved.
+
+An efficient approach to learning the materials is to first to skim
```

Comment:
`to first to skim`?

Reply:
Corrected, thank you! Yup I was thinking it'd be cool to drop all the edits at once in one pull request. Or is it possible to incorporate suggested edits in a pull request several times?